packages/server/graphql/private/typeDefs/_legacy.graphql

```diff
@@ -1049,397 +1049,6 @@ interface NewMeeting {
   locked: Boolean!
 }
 
-"""
```

Comment:
@nickoferrall I removed it in previous PR, looks like because of some merge issues, you added it back in your PR.
I removed it because now we merge public types with private. And instead of doing modifications in 2 places I just removed User type completely from here